src/Client.php

```diff
@@ -499,7 +500,15 @@ public function takeScreenshot($saveAs = null)
     {
         $this->start();
 
-        return $this->webDriver->takeScreenshot($saveAs);
+        if (null === $saveAs || (new Filesystem())->isAbsolutePath($saveAs)) {
+            return $this->webDriver->takeScreenshot($saveAs);
+        }
+
+        if (!($dir = $_SERVER['PANTHER_SCREENSHOT_DIR'] ?? false)) {
+            throw new \RuntimeException('$saveAs must be an absolute path if not using the PANTHER_SCREENSHOT_DIR env variable.');
```

Comment:
What's the point of throwing here? Shouldn't we keep the existing behavior? (save to the current wording directory)?

Actually, I wonder if we shouldn't keep the code in `Client` as-is and support this env var only for tests. We could rename the env var to `PANTHER_ERROR_SCREENSHOT_DIR` to be more explicit. WDYT?

Reply:
What about `Client::setScreenshotDir()` (see https://github.com/symfony/panther/pull/391#issuecomment-740593928). I think it could be useful for users to have a known directory for all the screenshots they manually take in their tests.

Reply:
But when using the standalone library (to create a web crawler for instance), saving relatively to the current working directory (as currently) looks like the obvious and expected behavior to me. Having to pass an absolute path or to set an env var is cumbersome.

Reply:
I also think this check makes no sense.
Also, I think one should be able to configure the client to *not* use the env var.
Can't we use an option? I see that the constructor takes an `$options` array.

Reply:
> Actually, I wonder if we shouldn't keep the code in Client as-is and support this env var only for tests. We could rename the env var to PANTHER_ERROR_SCREENSHOT_DIR to be more explicit. WDYT?

Maybe this is the best solution, @nicolas-grekas , wdyt?

Reply:
Lgtm